lib/HackTest.rb

```diff
@@ -56,4 +64,29 @@ def add_to_hash(found_words, word, file_name)
       found_words[word] = Set[file_name]
     end
   end
+
+  def handle_pull_request(pr_payload, non_inclusive_words_found)
+    remove_pr_comments(pr_payload)
+    add_pr_comment(pr_payload, 'Non inclusive words found:')
+  end
+
+  def remove_pr_comments(pr_payload)
+    comments = github_client.issue_comments(
+        pr_payload[:repository][:full_name],
+        pr_payload[:pull_request][:number])
+
+    comments.each do |comment|
+      github_client.delete_comment(
+          pr_payload[:repository][:full_name],
+          comment.to_h[:id])
+    end
```

Comment:
Won't this delete _all_ comments, not just the comments created by the bot? Is there a way to filter by author?